modules/storage-ephemeral-storage-manage.adoc

```diff
@@ -40,17 +58,24 @@ spec:
     image: images.my-company.example/log-aggregator:v6
     resources:
       requests:
-        ephemeral-storage: "2Gi" <1>
+        ephemeral-storage: "2Gi" <3>
     volumeMounts:
     - name: ephemeral
       mountPath: "/tmp"
   volumes:
     - name: ephemeral
       emptyDir: {}
 ----
-<1> Request for local ephemeral storage.
-<2> Limit for local ephemeral storage.
+<1> Container request for local ephemeral storage.
+<2> Pod request for local ephemeral storage.
+<3> Container request for local ephemeral storage.
+
+ifndef::microshift[]
+[id=storage-ephemeral-storage-scheduling-eviction_{context}]
+== Ephemeral storage configuration effects pod scheduling and eviction
+The settings in the pod spec affect both how the scheduler makes a decision about scheduling pods and when kubelet evicts pods.
 
-This setting in the pod spec affects how the scheduler makes a decision on scheduling pods, and also how kubelet evict pods. First of all, the scheduler ensures that the sum of the resource requests of the scheduled containers is less than the capacity of the node. In this case, the pod can be assigned to a node only if its available ephemeral storage (allocatable resource) is more than 4GiB.
+* First, the scheduler ensures that the sum of the resource requests of the scheduled containers is less than the capacity of the node. In this case, the pod can be assigned to a node only if the node's available ephemeral storage (allocatable resource) is more than 4GiB.
 
-Secondly, at the container level, since the first container sets resource limit, kubelet eviction manager measures the disk usage of this container and evicts the pod if the storage usage of this container exceeds its limit (4GiB). At the pod level, kubelet works out an overall pod storage limit by adding up the limits of all the containers in that pod. In this case, the total storage usage at the pod level is the sum of the disk usage from all containers plus the pod's `emptyDir` volumes. If this total usage exceeds the overall pod storage limit (4GiB), then the kubelet also marks the pod for eviction.
\ No newline at end of file
+* Second, at the container level, because the first container sets a resource limit, kubelet eviction manager measures the disk usage of this container and evicts the pod if the storage usage of the container exceeds its limit (4GiB). The kubelet also marks the pod for eviction if the total usage exceeds the overall pod storage limit (4GiB).
```

Comment:
```suggestion
* Second, at the container level, because the first container sets a resource limit, kubelet eviction manager measures the disk usage of this container and evicts the pod if the storage usage of the container exceeds its limit (4GiB). The kubelet also marks the pod for eviction if the total usage exceeds the overall pod storage limit (8GiB).
```

Reply:
@radeore We don't have the scheduler API in MicroShift, but does the kubelet eviction manager still operate on a MicroShift cluster?

Reply:
@ShaunaDiaz  I tested this scenario on Microshift and when ephemeral-storage usage of container exceeds resource: limits, pod does get 'Evicted' as expected. 
Hi @copejon , could you confirm if the above scenario is actually supported on Microshift?

Reply:
It's true that MicroShift doesn't include the `scheduler.config.openshift.io` API, but this should not impact whether pods are evicted when exceeding the default ephemeral storage quotas.  This policy is not applied to persistent storage (i.e. topolvm PVs) - it's strictly for `emptyDir` volumes.  Users can specify the pod's priorityClass to exempt specify pods from eviction.